repeated-integration-tests.bash

```diff
@@ -40,14 +42,14 @@ DIR="$(dirname "$(readlink -f "$0")")"
 runcount=0
 while [ "${runcount}" -lt "${runs}" ]; do
     timestamp="$(date -Iminutes)"
-    client_logfile="$DIR/target/cryostat-itests-${timestamp}.client.log"
-    server_logfile="$DIR/target/cryostat-itests-${timestamp}.server.log"
+    client_logfile="$DIR/target/${POD_NAME}-${timestamp}.client.log"
```

Comment:
I noticed when all the itests finish running, the test results are overwritten with the following. Where did the previous logs go?
`/home/jalaw/cryostat/target/cryostat-itests-2021-12-02T18:30-05:00.client.log`
```
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------------------< io.cryostat:cryostat >------------------------
[INFO] Building cryostat 2.1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (stop-jfr-datasource) @ cryostat ---
jfr-datasource-itest
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (stop-grafana) @ cryostat ---
grafana-itest
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (stop-container) @ cryostat ---
cryostat-itest
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (destroy-pod) @ cryostat ---
83c9173a91e43a40b7515c5f1532302d981a1455d17ddd3c9bc2b286ad5c73e0
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.403 s
[INFO] Finished at: 2021-12-02T18:34:03-05:00
[INFO] ------------------------------------------------------------------------
```

Reply:
Whoops. My "pipecleaner" change with the optionally preserved coloured output seems to have broken that. I think I fixed it.